stustapay/core/service/customer/customer.py

```diff
@@ -148,7 +148,7 @@ async def get_payout_transactions(self, *, conn: Connection, current_customer: C
             PayoutTransaction,
             "select t.amount, t.booked_at, a.name as target_account_name, a.type as target_account_type, t.id as transaction_id "
             "from transaction t join account a on t.target_account = a.id "
-            "where t.order_id is null and t.source_account = $1 and t.target_account in (select id from account where type = 'cash_exit' or type = 'donation_exit')",
```

Comment:
Cash exit still is a valid account type since we have cash payouts during the event